packages/notebook-extension/src/index.ts

```diff
@@ -3536,10 +3536,8 @@ function addCommands(
   });
 
   commands.addCommand(CommandIDs.virtualScrollbar, {
-    label: trans.__('Show Virtual Scrollbar'),
-    caption: trans.__(
-      'Show virtual scrollbar (enabled with windowing mode: full)'
-    ),
+    label: trans.__('Show Minimap'),
+    caption: trans.__('Show Minimap (enabled with windowing mode: full)'),
```

Comment:
I wonder if it makes sense to mention both minimap and virtual scrollbar in caption for continuity? Otherwise I think it is a great idea!

Reply:
Sound good. So keeping `Show Minimap` for the `label`, and something like `Show Minimap (virtual scrollbar, enabled with windowing mode: full)` for the `caption`?

Reply:
Sounds good to me!